packages/mock-block-dock/dev/dev-app.tsx

```diff
@@ -15,16 +15,34 @@ type TestBlockType =
   | "html-at-url"
   | "html-as-string";
 
+const blockEntityMap = {
+  react: {
+    entityId: "entity-react",
+    entityTypeId: "test-type-1",
+    properties: { name: "World" },
+  },
+  "custom-element": {
+    entityId: "entity-custom-element",
+    properties: { name: "World" },
+  },
+  "html-at-url": {
+    entityTypeId: "test-type-1",
+    properties: { name: "World" },
+  },
+  "html-as-string": {
+    entityId: "entity-html-at-url",
+    entityTypeId: "test-type-1",
+    properties: { name: "World" },
+  },
```

Comment:
Added these to test scenarios where any or both of entityId and entityTypeId are absent to be sure it doesn't break